x11-libs/qscintilla/qscintilla-2.13.3.ebuild

```diff
@@ -56,24 +73,41 @@ qsci_run_in() {
 }
 
 src_configure() {
-	if use designer; then
-		# prevent building against system version (bug 466120)
-		append-cxxflags -I../src
-		append-ldflags -L../src
-	fi
+	my_src_configure() {
+		if use designer; then
+			# prevent building against system version (bug 466120)
+			append-cxxflags -I../src
+			append-ldflags -L../src
+		fi
+
+		local QT_VERSION=$(echo ${MULTIBUILD_VARIANT} | cut -c3)
+		case ${MULTIBUILD_VARIANT} in
+			qt5) local QMAKE=eqmake5 ;;
+			qt6) local QMAKE=qmake6 ;;
+		esac
+		qsci_run_in "${BUILD_DIR}"/src "${QMAKE}"
+		use designer && qsci_run_in "${BUILD_DIR}"/designer "${QMAKE}" INCLUDEPATH+="${BUILD_DIR}"/src LIBS+=-L"${BUILD_DIR}"/src
```

Comment:
can you explain why INCLUDEPATH and LIBS are needed?

Reply:
good catch - isn't needed